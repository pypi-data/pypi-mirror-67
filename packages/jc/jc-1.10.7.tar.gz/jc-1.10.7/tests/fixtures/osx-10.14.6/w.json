[{"user": "kelly", "tty": "console", "from": null, "login_at": "22Nov19", "idle": "20days", "what": null}, {"user": "kelly", "tty": "s000", "from": null, "login_at": "Wed16", "idle": null, "what": "ssh localhost"}, {"user": "kelly", "tty": "s001", "from": "127.0.0.1", "login_at": "10:14", "idle": null, "what": "-bash"}, {"user": "kelly", "tty": "s002", "from": null, "login_at": "10:14", "idle": null, "what": "ssh 127.0.0.1"}, {"user": "kelly", "tty": "s003", "from": "127.0.0.1", "login_at": "10:14", "idle": null, "what": "-bash"}, {"user": "kelly", "tty": "s004", "from": null, "login_at": "10:14", "idle": null, "what": "ssh ::1"}, {"user": "kelly", "tty": "s005", "from": "::1", "login_at": "10:15", "idle": null, "what": "-bash"}, {"user": "kelly", "tty": "s006", "from": null, "login_at": "10:15", "idle": null, "what": "w"}]
