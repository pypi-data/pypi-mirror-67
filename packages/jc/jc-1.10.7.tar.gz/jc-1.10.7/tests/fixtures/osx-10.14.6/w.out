10:15  up 20 days, 19:44, 8 users, load averages: 0.86 0.45 0.37
USER     TTY      FROM              LOGIN@  IDLE WHAT
kelly    console  -                22Nov19 20days -
kelly    s000     -                Wed16       - ssh localhost
kelly    s001     127.0.0.1        10:14       - -bash
kelly    s002     -                10:14       - ssh 127.0.0.1
kelly    s003     127.0.0.1        10:14       - -bash
kelly    s004     -                10:14       - ssh ::1
kelly    s005     ::1              10:15       - -bash
kelly    s006     -                10:15       - w
